backend/dao/profileDAO.js

```diff
@@ -39,7 +39,7 @@ export default class ProfileDAO {
         { _id: new mongoose.Types.ObjectId(clientID) },
         { $set: profileDoc }
       );
-      return updateResponse;
```

Comment:
Don't we need to return the response to the frontend here? Might want to add this back.

Reply:
Because the front end stores the user's values and then sends them to the back end, we never used the values being returned. I can add it back if you still think we should, I just took it off to increase code coverage

Reply:
Alrighty, no worries then.